pkg/controller/endpointslice/utils.go

```diff
@@ -166,6 +166,11 @@ func newEndpointSlice(service *corev1.Service, endpointMeta *endpointMeta) *disc
 		AddressType: endpointMeta.AddressType,
 		Endpoints:   []discovery.Endpoint{},
 	}
+	// add parent service labels
+	for key, value := range service.Labels {
+		eps.Labels[key] = value
+	}
```

Comment:
This should follow the pattern in EndpointSliceMirroring controller, need to be careful to ensure that labels on Service can't override important EndpointSlice labels: https://github.com/kubernetes/kubernetes/blob/90723ebbae0753a08a34aa10e7bc5efc98b2cf5a/pkg/controller/endpointslicemirroring/utils.go#L100-L105

Reply:
done